llvm/lib/CodeGen/CFIInstrInserter.cpp

```diff
@@ -272,7 +272,7 @@ void CFIInstrInserter::calculateOutgoingCFAInfo(MBBCFAInfo &MBBInfo) {
           CSRLocMap.insert(
               {CFI.getRegister(), CSRSavedLocation(CSRReg, CSROffset)});
         } else if (It->second.Reg != CSRReg || It->second.Offset != CSROffset) {
-          llvm_unreachable("Different saved locations for the same CSR");
+          report_fatal_error("Different saved locations for the same CSR");
```

Comment:
I think this should be `reportFatalInternalError` - I'm not sure this line is achievable without a MIR test, but I'm not sure.